### PR TITLE
Created account count

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Api.Worker/MessageConsumers/AccountAddedConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            _logger = logger;
+            _brokerAccountRepository = brokerAccountRepository;
+        }
+
+        public async Task Consume(ConsumeContext<AccountAdded> context)
+        {
+            var evt = context.Message;
+
+            var brokerAccount = await _brokerAccountRepository.GetAsync(evt.BrokerAccountId);
```
Comment (KonstantinRyazantsev) [inline src/Api.Worker/MessageConsumers/AccountAddedConsumer.cs:27]: we even could do this without reading the broker account. We can increment accounts count on the DB side using EF Plus Update method:

```c#
                .UpdateAsync(x => new BrokerAccount
                    {
                        AccountCount = x.AccountCount + 1
                    });
```

File: src/Api/WebApi/BrokerAccountController.cs [hunk 2]
```diff
@@ -416,7 +417,8 @@ private static BrokerAccountResponse MapToResponse(BrokerAccount brokerAccount)
                     _ => throw new ArgumentOutOfRangeException(nameof(brokerAccount.State), brokerAccount.State, null)
                 },
                 CreatedAt = brokerAccount.CreatedAt,
-                UpdatedAt = brokerAccount.UpdatedAt
+                UpdatedAt = brokerAccount.UpdatedAt,
+                AccountCount = brokerAccount.AccountCount
```
Comment (KonstantinRyazantsev) [inline src/Api/WebApi/BrokerAccountController.cs:421]: `AccountsCount`

File: src/Api.Common/ReadModels/BrokerAccounts/BrokerAccount.cs [hunk 3]
```diff
@@ -10,5 +10,6 @@ public class BrokerAccount
         public DateTimeOffset CreatedAt { get; set; }
         public DateTimeOffset UpdatedAt { get; set; }
         public BrokerAccountState State { get; set; }
+        public long AccountCount { get; set; }
```
Comment (KonstantinRyazantsev) [inline src/Api.Common/ReadModels/BrokerAccounts/BrokerAccount.cs:13]: `AccountsCount`

File: src/Api.Worker/MessageConsumers/AccountAddedConsumer.cs (hunk 1 above)
Comment (OttoVT) [reply src/Api.Worker/MessageConsumers/AccountAddedConsumer.cs:27]: Nope =)

**Pull request merged**